### PR TITLE
fix missing reshares count on mobile stream #5693

### PR DESCRIPTION
**Pull request by visualsayed**

### CODE REVIEW

**Comment by jhass** [comment]
You're right, the `.post_stats` seem duplicated, extracting them to a partial like `app/views/posts/_post_stats.mobile.haml` would be nice.

**Comment by visualsayed** [comment]
Thanks, i will do it.

**Comment by visualsayed** [comment]
@jhass I think it is done. thanks

File: app/views/shared/_stream_element.mobile.haml [hunk 1]
```diff
@@ -30,22 +30,5 @@
...
-          %span.like_count
-            = post.likes.size
-
-        %ul.comments
-          = render :partial => 'comments/comment', :collection => post.comments.order('created_at ASC')
-
-          %li.comment.add_comment_bottom_link_container
-            - if user_signed_in? 
-              = link_to t('comments.new_comment.comment'), new_post_comment_path(post), :class => 'add_comment_bottom_link btn comment_action inactive'
+      = render :partial => 'shared/post_stats', :locals => {:post => @post}
```
Comment (jhass) [inline app/views/shared/_stream_element.mobile.haml]: Please update to the Ruby 1.9 hash style, that we prefer as per our new [styleguide](https://wiki.diasporafoundation.org/Styleguide): `render partial: "shared/post_stats", locals: {post: @post}`

File: app/views/shared/_post_stats.mobile.haml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      = @post.comments.size
+    %span.like_count
+      = @post.likes.size
+
+  %ul.comments
+    = render :partial => 'comments/comment', :collection => @post.comments, :locals => {:post => @post}
+
+    %li.comment.add_comment_bottom_link_container
+      = link_to "#", :class => "show_comments bottom_collapse active" do
+        = image_tag 'icons/arrow_up_small.png'
```
Comment (jhass) [inline app/views/shared/_post_stats.mobile.haml]: While touching this code please update it to our new [styleguide](https://wiki.diasporafoundation.org/Styleguide) by preferring double over single quotes.

**Comment by visualsayed** [comment]
You konw what i thought that you prefere the opposite :-), Right now i will do it.

**Comment by visualsayed** [comment]
done.

**Comment by jhass** [comment]
Thanks!

**Pull request merged**